contracts/DCAStrategies/DCAStrategies/DCAStrategiesPositionsHandler.sol

```diff
@@ -7,42 +7,34 @@ import '../../interfaces/IDCAStrategies.sol';
 abstract contract DCAStrategiesPositionsHandler is IDCAStrategiesPositionsHandler {
   using SafeERC20 for IERC20;
 
-  // TODO: add function similar to this one https://github.com/Mean-Finance/dca-v2-core/blob/main/contracts/interfaces/IDCAHub.sol#L243
+  mapping(uint256 => Position) internal _userPositions;
 
   /// @inheritdoc IDCAStrategiesPositionsHandler
   function deposit(IDCAStrategies.DepositParams calldata _parameters) external returns (uint256) {
```

Comment:
Let's return the created positions as part of this function. Since we already have them, why not?

Reply:
Which function are you referring to?

Reply:
Ignore me haha, it's been a long day and I don't like the idea anymore. Let's keep this like it is